### ISSUE TITLE
tldr for own custom scripts?

### ISSUE THREAD

**Issue by tmarplatt** [raylee/tldr#28]
Hey again, how should one go about creating a custom .md file that just doesn't qualify for the public repo?

Say I have a custom shell script I wrote and use twice a month (and I never remember the parameters), tldr would be the perfect tool to just remind myself of the script's usage.

Can I add my own pages, currently? Won't they get deleted from the cache?

Edit: So it turns out adding my own .md under /common doesn't work, possibly because tldr looks up the online repo and decides the command doesn't exist.

**Comment by raylee** [comment]
Yeah, tldr is looking in the index.json file first for the page. I've just pushed an update to look for the page in $configdir/local/ first. The commit message contains details:  https://github.com/raylee/tldr/commit/92e0545048b9bb702aca62b7df30fe299defb5c9
tldr will not touch those pages when updating.

**Comment by raylee** [comment]
If the new version doesn't work for you, please re-open this issue. Thanks!
